build/build/src/enso.rs

```diff
@@ -1,6 +1,6 @@
 use crate::prelude::*;
 
-use crate::paths::Paths;
+use crate::paths::{ENSO_ENABLE_ASSERTIONS, Paths};
```

Comment:
Shouldn't this be
```suggestion
use crate::paths::Paths;
use crate::paths::ENSO_ENABLE_ASSERTIONS;
```
?
Looking at the lines below, I thought the statements are split each to separate line on purpose.